starter-code/js/application.js

```diff
@@ -1,8 +1,99 @@
-// Use this file to write the interactions between your game and the user
+window.onload = function () {
+    var answer = "",
+        interval,
+        audio = document.createElement("audio");
+    
+    //play sound
+    function sound() {
+        audio.setAttribute("src", "./audios/cash-register.mp3");
+        audio.setAttribute("autoplay", "autoplay");
+    }
 
-//Initialize ion library
+    //change the input number of the option Number Limit 
+    $('#range-slider').on("change mousemove", function () {
+        $('#number').html(this.value);
+    });
 
+    //check for the user operator selection
+    $("input").on("click", function () {
+        if ($("#" + this.id).prop("checked")) {
+            if (this.id === "add") {
+                $('#operator').html("+");
+                quest();
+            }
+            else if (this.id === "sub") {
+                $('#operator').html("-");
+                quest();
+            }
+            else if (this.id === "multi") {
+                $('#operator').html("*");
+                quest();
+            }
+            else if (this.id === "div") {
+                $('#operator').html("/");
+                quest();
+            }
+        }
+    });
 
-window.onload = function(){
+    //set the correct answer to a global variable
+    function result(num1, num2) {
+        correctAnswer = eval(num1 + $('#operator').html() + num2);
+    }
+
+    //random 2 numbers
+    function quest() {
+        var number = $('#number').html(),
+            num1 = Math.floor(Math.random() * (number - 1) + 1),
+            num2 = Math.floor(Math.random() * (number - 1) + 1);
+        $('#number1').html(num1);
+        $('#number2').html(num2);
+        result(num1, num2);
+    }
+
+    //hide the first screen and start the game
+    $(".start").on("click", function () {
+        $(".setting-panel").hide();
+        $(".game").css("visibility", "visible");
+        startgame();
+    });
+
+    //restart the game
+    $(".restart").on("click", function () {
+        clearInterval(interval);
+        $('#countdown').html(10);
```

Comment:
the value of the seconds should not be grabbed from the html, the values that belong to the logic should be set up and managed in the logic, when I say logic I refer to functions or objects that handle the behavior of the game. A good way to handle the data is to build an object game that has properties and methods, a property could be timer and based on the result of functions such checkAnswer that property will increse by ten or not as it's suppose to do.
Good job anyway